packages/react/src/views/Message/Message.styles.js

```diff
@@ -75,7 +75,7 @@ export const getMessageDividerStyles = (theme) => {
       line-height: 1rem;
       position: relative;
       display: flex;
-      z-index: 1000;
+      z-index: 1;
```

Comment:
Is there any specific reason for changing this? The current structure uses a specific set of z-index values for different elements, so it behaves correctly during integration. Is it necessary to change it? Please look for ways to avoid this change

Reply:
The z-Index is made 1 so that the date for the messages does not appear over sidebar menu when it gets adjusted to Full Screen